systemtests/README.rst

```diff
@@ -32,31 +32,26 @@ environments.
 Running tests
 =============
 
-First, make sure you have valid, unexpired API tokens for the environment
-you're testing.
+The system tests are run using the ``test_ven.py`` Python script. You can get
+help about the command-line invocation of that script using::
 
-For destructive tests run in local and stage, you need separate auth tokens for
-try uploads with "Upload Try Symbols Files" permissions. See Bug 1759740.
+    $ make shell
+    root@e62fb7ae586f:/app# cd systemtests
+    root@e62fb7ae586f:/app/systemtests# ./test_env.py --help
 
-To set auth tokens, add these to your .env file:
+The help includes a list of available environments.
 
-* `LOCAL_AUTH_TOKEN`
-* `LOCAL_AUTH_TOKEN_TRY`
-* `STAGE_AUTH_TOKEN`
-* `STAGE_AUTH_TOKEN_TRY`
-* `PROD_AUTH_TOKEN`
+You need to make sure you have valid, unexpired API tokens for the environment
+you're testing. Add these tokens to your ``.env`` file using the environment
+variable names in the help output, e.g. ``STAGE_AUTH_TOKEN`` and
+``STAGE_AUTH_TOKEN_TRY`` for the stage environment.
 
-To run the systemtests, do::
-
-   $ make shell
-   root@f09b3cdf8570:/app# cd systemtests/
-   root@e62fb7ae586f:/app/systemtests# ./test_env.py ENVIRONMENT
+For destructive tests run in local and stage, you need separate auth tokens for
+try uploads with "Upload Try Symbols Files" permissions. See Bug 1759740.
 
-where ``ENVIRONMENT`` is one of the following:
+To run the systemtests, do::
 
-* ``local``: run all tests against your local dev environment
-* ``stage``: run all tests against stage
-* ``prod``: run non-destructive tests against prod
+    root@e62fb7ae586f:/app/systemtests# ./test_env.py <ENV_NAME>
```

Comment:
This assumes you've done a bunch of setup work now listed under `USAGE` in the `test_env.py` `--help`. Why not keep those in sync, else reference `USAGE` in the `--help` here?

Reply:
The setup is documented in the first section of the README file. The first section is "Setting up the tests". The second section is "Running the tests".